content/docs/application-architecture/_index.md

```diff
@@ -80,7 +80,11 @@ To mitigate integrations issue between the User Portal and other services,
 
 #### Authentication
 
-Responsible to connect to the AAI and IDP, to authenticate and retrieve authorisation. It will be implemented via OIDC, following the GA4GH passport specification. The token will be shared among all the components.
+Responsible for validating that a user corresponds to the expected person. This is achieved through the OIDC protocol, allowing users to login using their host organisations. The confidence that a logged-in user corresponds to the expected person depends on the assurance level of the idP of used to authenticate the current session. The relevant assurance levels are defined in the [eIDAS Regulation](http://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv%3AOJ.L_.2014.257.01.0073.01.ENG).
+
+#### Authorization
+
+This component is responsible for validating the different authorization of a logged-in user. This is achieved by following the GA4GH Passport specification, and sharing passports between components. Upon access to a restricted resource, the passport granting access is validated at the authority granting the authorization, to ensure that the passport has not been revoked during the current session.
```

Comment:
@hallundbaek thanks for your update. There are two different approaches of AuthZ for two different kind of users, on GDI User Portal context.

GA4GH visas for Beacon Network query, and possibly to evaluate if the user has requested access to a dataset already.
And role-based permission in CKAN, to manage access to CKAN features, like harvester and manual input of datasets.

Perhaps we should mention both scenarios, what do you think?

Reply:
@brunopacheco1 who decides the roles in CKAN? On what authority are these awarded?

Assuming the target audience of this doc are the nodes that need to feed into the user-portal, these are the important things to know:

1. What are authorisations can a user have
2. For each authorisation:
  a. What does the authorisation authorise (CKAN role? dataset access? ...)
  b. Who is the authority of the authorisation
  c. How does the user-portal retrieve this authorisation
  d. How does the user-portal guarantee that authorisation is valid at the time of use

Reply:
@hallundbaek this governance is not defined yet, but should be addressed in the "who will operate the portal" discussions over the coming years. For now, LU is responsible for the UP, so it could be LU until this is all defined. We can add it to the WP4 discussion / PILLAR II discussion list in coming months?

Reply:
@hallundbaek Thanks again for your suggestions. I will merge you changes and later add more details on protocols and specifications the User Portal will implement.

Anyway, just to clarify, the Application Architecture is a document that records important facts and serve as guidelines for the User Portal development, it is not written to support nodes. For that further documentation is required.

As @Trefex mentioned, this governance is yet to be defined. But, given the need to maintain datasets metadata in multiple different systems, I hardly believe any node will be feeding datasets direct into CKAN, although there is no formal decision on that (as far as I know).